### ISSUE TITLE
Fail to launch as a system service with error code 203/exec

### ISSUE THREAD

**Issue by Blade-B3AR** [Spotifyd/spotifyd#431]
when i ever i go to start it with systemctl start i get error code 203/exec
would like some help i am trying to use it with spotify-tui if that helps
also using zsh instead of bash

here is my spotifyd.service file

[Unit]
Description=A spotify playing daemon
Documentation=https://github.com/Spotifyd/spotifyd

[Service]
ExecStart=/etc/systemd/system/spotifyd.service --no-daemon
Restart=always
RestartSec=12

[Install]
WantedBy=default.target

❯ sudo systemctl status spotifyd.service 
● spotifyd.service - A spotify playing daemon
   Loaded: loaded (/etc/systemd/system/spotifyd.service; enabled; vendor preset: disabled)
   Active: activating (auto-restart) (Result: exit-code) since Sat 2019-11-09 20:30:03 CST; 5s ago
     Docs: https://github.com/Spotifyd/spotifyd
  Process: 7145 ExecStart=/etc/systemd/system/spotifyd.service --no-daemon (code=exited, status=203/EXEC)
 Main PID: 7145 (code=exited, status=203/EXEC)

~
❯

**Comment by SirWindfield** [comment]
https://stackoverflow.com/questions/45776003/fixing-a-systemd-service-203-exec-failure-no-such-file-or-directory

**Comment by SirWindfield** [comment]
https://unix.stackexchange.com/questions/472950/systemd-status-203-exec-error-when-creating-new-service

**Comment by awerlang** [comment]
ExecStart is pointing to the service definition, not the binary

**Comment by afritsch89** [comment]
Ok what about mine?

[Unit]
Description=A spotify playing daemon
Documentation=https://github.com/Spotifyd/spotifyd
Wants=sound.target
After=sound.target
Wants=network-online.target
After=network-online.target

[Service]
ExecStart=/usr/bin/spotifyd
Restart=always
RestartSec=12

[Install]
WantedBy=default.target


That is where the binary is, and when I run it manually I can make it work, but I'm getting a unit not found error.

**Comment by awerlang** [comment]
@afritsch89 try this .service https://github.com/Spotifyd/spotifyd/blob/0efe0fee3f7ad9fab77018f0bf58da199b381842/contrib/systemd/user/spotifyd.service from #600

**Comment by SirWindfield** [comment]
Pretty sure you should pass down `no-daemon`: `ExecStart=/usr/bin/spotifyd --no-daemon`. That way you get log messages.

**Comment by afritsch89** [comment]
Sorry, half asleep when I posted that.

I really should start up a new issue for this but,

I originally used the pre packaged spotifyd.service

Got the error `Failed to start spotifyd.service.Unit spotifyd.service not found`

Tried `systemctl daemon-reload` but no dice.

I'm thinking this is more of a systemd issue so I'll poke in that direction, but if anyone else has seen this and knows the fix that would be dope.
